src/Sarif.Converters/StaticDriverVerifierConverter.cs

```diff
@@ -228,14 +265,25 @@ private void ProcessLine(string logFileLine, Result result)
             }
         }
 
-        private string BuildMessageFromState(string state)
+        private bool IsHarnessOrRulesFiles(string fileName)
         {
-            if (string.IsNullOrEmpty(state) || state == "^")
+            return fileName.EndsWith(".slic") || fileName.EndsWith("sdv-harness.c");
```

Comment:
Do these need to be case insensitive? Esp the file extension since that could be user code (for a custom rule.)